### PR TITLE
Visual Studio Code docs

### PR DESCRIPTION
**Pull request by pomverte**

### CODE REVIEW

File: VisualStudioCode/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Visual Studio Code
+
+[Visual Studio Code](https://code.visualstudio.com/) is a lightweight code editor with support for many programming langagues through [extensions](https://code.visualstudio.com/docs/editor/extension-gallery)
+
+## Installation
+
+To install the latest version, use homebrew:
```
Comment (hugovk) [inline VisualStudioCode/README.md]:
```suggestion
To install the latest version, use Homebrew:
```

File: VisualStudioCode/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+```bash
+brew cask install visual-studio-code
+```
+
+## MacOS integration
```
Comment (hugovk) [inline VisualStudioCode/README.md]:
```suggestion
## macOS integration
```

File: VisualStudioCode/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+Launch VSCode from [command line](https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line)
```
Comment (hugovk) [inline VisualStudioCode/README.md]:
```suggestion
Launch VS Code from the [command line](https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line).
```

File: VisualStudioCode/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+After that, you can launch VSCode from your terminal :
```
Comment (hugovk) [inline VisualStudioCode/README.md]:
```suggestion
After that, you can launch VS Code from your terminal:
```

File: VisualStudioCode/README.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+* `code .` will open VSCode in the current directory
```
Comment (hugovk) [inline VisualStudioCode/README.md]:
```suggestion
* `code .` will open VS Code in the current directory
```

File: VisualStudioCode/README.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+* `code myfile.txt` will open `myfile.txt` in VSCode
```
Comment (hugovk) [inline VisualStudioCode/README.md]:
```suggestion
* `code myfile.txt` will open `myfile.txt` in VS Code
```

**Comment by hugovk** [comment]
Should this also be listed in https://github.com/sb2nov/mac-setup/blob/master/SUMMARY.md?

**Comment by pomverte** [comment]
> Should this also be listed in https://github.com/sb2nov/mac-setup/blob/master/SUMMARY.md?

haha surely, thanks for the review 👍

**Comment by pomverte** [comment]
Hello !
Any news on this PR ? How can this be pulled from now ?
Thanks !

**Comment by hugovk** [comment]
Nobody objected, let's merge it. Thanks!

**Pull request merged**